tests/ui_tests/app/page/common/configuration/page.py

```diff
@@ -70,6 +73,17 @@ def get_input_value(self, adcm_test: str, is_password: bool = False):
         )
         return self.find_element(template(adcm_test)).get_property("value")
 
+    @allure.step('Check input of field "{adcm_test}" has value "{expected_value}"')
+    def assert_input_value_is(self, expected_value: str, adcm_test: str, is_password: bool = False):
```

Comment:
Why adcm_test is a string and not a webElement?

Reply:
It's a string that is passed to format TemplateLocator to generate an actual Locator.
It follows the signature of `get_input_value` method of the Configuration page.